### DISCUSSION TITLE
Should `set-language` be renamed to `language`?

### DISCUSSION THREAD

**Discussion by Jummit** [helix-editor/helix#7810] [Ideas] [answered]
Most commands that can be used to retrieve or change a setting (`theme`, `indent-style`, `encoding`) don't have the `set-` prefix. This makes sense, as they without parameters they just give you the current state.

So, should `set-language` be renamed to `language`?

I think it would be more convenient and consistent. What do you think?

**Comment by pascalkuthe** [comment, marked as answer]
we don't rename commands usually unless absolutely necessary (since it breaks configs) so we won't rename it. There is already a `:lang` alias

**Reply by Jummit** [reply]
That makes sense. I somehow overlooked the alias, thanks for the tip!
